trustregistry/registry/registry_schemas.py

```diff
@@ -95,12 +97,28 @@ async def update_schema(
     return update_schema_res
 
 
+@router.get("/{schema_id}", response_model=Schema)
+async def get_schema(schema_id: str, db_session: Session = Depends(get_db)) -> Schema:
+    bound_logger = logger.bind(body={"schema_id": schema_id})
+    bound_logger.info("GET request received: Fetch schema")
+    try:
+        schema = crud.get_schema_by_id(db_session, schema_id=schema_id)
+    except crud.SchemaDoesNotExistException:
+        bound_logger.info("Bad request: Schema not found.")
+        raise HTTPException(
+            status_code=404,
+            detail="Schema not found.",
```

Comment:
detail message to user can be more verbose: `f"Schema with id {schema_id} not found."`

Reply:
Note: this adjusts expected messaged in pytests (can simply be changed to assert "not found" in response, instead of "Schema not found")